distributed/scheduler.py

```diff
@@ -385,12 +387,81 @@ def mark_ready_to_run(self, key):
                 self.unrunnable.add(key)
             else:
                 self.stacks[new_worker].append(key)
-                self.maybe_ready.add(new_worker)
+                self.maybe_idle.add(new_worker)
                 # self.ensure_occupied(new_worker)
         else:
             self.ready.appendleft(key)
             # self.ensure_idle_ready()
 
+    def should_steal(self, key, bandwidth=100e6):
```

Comment:
Maybe using a module level constant and/or an environment variable would allow people with high end hardware to tune this to better reflect their empirically observed bandwidth.